frontend/playground-frontend/src/components/Feedback/Feedback.tsx

```diff
@@ -2,7 +2,10 @@
 import React, { useState } from "react";
 import type { ReactNode } from "react";
 
-import { EmailInput, TitleText, Spacer } from "../index";
+import EmailInput from "../Home/EmailInput";
+import TitleText from "../general/TitleText";
+import Spacer from "../general/Spacer";
+//import { EmailInput, TitleText, Spacer } from "../index";
```

Comment:
we actually don't need this line here b/c it was causing the Feedback.test.tsx unit tests to fail